ktxspec.adoc

```diff
@@ -1516,13 +1539,14 @@ hardware vendors and open source developers.
 Formats are identified by one or more of the following:
 
 * A Vulkan VkFormat enum value.
-* A color model value from the Khronos Data Format Specification
-  <<KDF13>>.
+* A color model value from Section
```

Comment:
s/Section/Sections

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/224/commits/c21466eaa9f2f02cb19c9eec6b5d3eb86d99d285.